addons/web/static/src/core/errors/error_dialogs.js

```diff
@@ -33,8 +35,15 @@ export class ErrorDialog extends Dialog {
         this.state = useState({
             showTraceback: false,
         });
+        this.popover = usePopover();
     }
     onClickClipboard() {
+        // show copied popover and close it after 1 second so user can know text is copied
+        class Comp extends Component { }
```

Comment:
declare the component outside of the ErrorDialog Class.
This is specifically crucial for the `xml` function whcih adds a template in Qweb, but we really want that to happen only once.